nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/pom.xml

```diff
@@ -131,6 +136,11 @@
             </exclusions>
         </dependency>
         <dependency>
+            <groupId>org.apache.orc</groupId>
+            <artifactId>orc-core</artifactId>
```

Comment:
This JAR appears to have a NOTICE file that I don't believe has been included in our NOTICE.